libs/entry-components/styles/modules/vendors/rich-text/necessary-mix/_typography-generator.scss

```diff
@@ -0,0 +1,78 @@
+/* stylelint-disable selector-max-compound-selectors, scss/at-if-no-null */
+@use 'sass:map';
+@use '@angular/material' as mat;
+
+$body-typography: map.get(mat.define-typography-config(), 'body-1');
+$hero-typography: map.get(mat.define-typography-config(), 'headline-1');
+$title-typography: map.get(mat.define-typography-config(), 'subtitle-1');
+$caption-typography: map.get(mat.define-typography-config(), 'caption');
+
+@mixin generate-from($theme) {
+	$theming-fonts: map.get($theme, 'general', 'fonts');
+
+	.enigmatry-redactor-content {
+		.rx-editor, .rx-content, .rx-content p {
+			@if $theming-fonts and map.has-key($theming-fonts, 'body') and map.get($theming-fonts, 'body', 'color') != null {
```

Comment:
I attempted various approaches to avoid the null value and bypass the stylelint rule, but unfortunately, none of them were successful.